app/scripts/services/logLinks.js

```diff
@@ -73,7 +73,13 @@ angular.module('openshiftConsole')
       ].join(''));
 
 
-      var archiveUri = function(opts) {
+      var archiveUri = function(opts, prefix) {
+        var index = 'project.' + opts['namespace'] + '.' + opts['namespaceUid'] + '.*';
+        if(prefix) {
```

Comment:
Maybe we can reduce some noise with `opts.namespace` & `opts.namespaceUid`, rather than array syntax?